### ISSUE TITLE
Scaling Skeleton with PhysicalBones results in math errors

### ISSUE THREAD

**Issue by arnerak** [godotengine/godot#40614]
**Godot version:**
3.2.1

**OS/device including version:**
Windows 10 1903

**Issue description:**
This error appears (many times) when scaling up a rigged character with PhysicalBones.
*core/math/basis.cpp:304 - Condition "!m.is_diagonal()" is true. Returned: Vector3()*
In a project of mine these errors already appear in the editor, when switching to a scene tab that contains such a Node. It also happens when running the game once the scene is initalized, as well as when closing the game. In the attached project it only happens when closing the game for some reason. It does not matter whether I scale up the character in blender or godot.
Note that this also breaks HingeJoints:
![gg](https://user-images.githubusercontent.com/29173710/88230660-18f16500-cc73-11ea-8241-997866e087b6.gif)

**Steps to reproduce:**
  1. Let mixamo rig a model
  2. import into blender
  2.1. (scale up armature and mesh in blender)
  3. export better collada
  4. auto generate physical bones
  5. (scale up if 2.1 was skipped)
  6. run physical bone simulation

**Minimal reproduction project:**
[godottest.zip](https://github.com/godotengine/godot/files/4962696/godottest.zip)

**Comment by fire** [comment]
Is the scaling uniform or not? Like 2x on all axes?

**Comment by arnerak** [comment]
> 
> 
> Is the scaling uniform or not? Like 2x on all axes?

Yes, however it only tends to glitch out from large scaling factors like 10 and above. 
For now I have scaled the mesh properly before importing it into mixamo to avoid scaling it afterwards and it works fine.
